packages/main/src/CardHeader.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="{{classes}}"
+     @click="{{_headerClick}}"
+     @keydown="{{_headerKeydown}}"
+     @keyup="{{_headerKeyup}}"
+     role="{{ariaHeaderRole}}"
+     aria-labelledby="{{ariaLabelledByHeader}}"
+     aria-level="{{ariaLevel}}"
+     aria-roledescription="{{ariaCardHeaderRoleDescription}}"
+     tabindex="0"
+    id="{{_id}}--header">
```

Comment:
bad tabulation
Also, the best practice is to always put the closing ">" on a new line, in order to achieve better diffs and easier conflict resolution when merging code.

Reply:
done